data/json/regional_map_settings.json

```diff
@@ -523,55 +523,55 @@
       "field": {
         "chance": 6,
         "extras": {
-          "mx_helicopter": 1,
-          "mx_military": 2,
-          "mx_science": 1,
-          "mx_collegekids": 3,
-          "mx_drugdeal": 3,
-          "mx_supplydrop": 4,
-          "mx_portal": 1,
-          "mx_crater": 15,
-          "mx_portal_in": 1,
-          "mx_point_dead_vegetation": 50,
-          "mx_grass": 350,
-          "mx_grass2": 350,
-          "mx_trees": 50,
-          "mx_trees2": 80,
-          "mx_Trapdoor_spider_den": 4,
-          "mx_fallen_shed": 20,
-          "mx_pond": 20,
-          "mx_point_burned_ground": 50,
-          "mx_casings": 20,
-          "mx_corpses": 3,
-          "mx_nest_wasp": 5,
-          "mx_mass_grave": 5,
-          "mx_grave": 5,
-          "mx_exocrash_1": 1,
-          "mx_exocrash_2": 1
+          "mx_helicopter": 10,
```

Comment:
This should stay less frequent than mx_military.

Reply:
what values do you suggest then?

Reply:
2 would keep it the same ratio, but 1 is also acceptable.